### PR TITLE
created basic components for scoreboard and profile, added navigation

### PR DESCRIPTION
**Pull request by pncifra**
Base components created. Added navigation and links to the new components.

### CODE REVIEW

File: src/components/Profile.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
```
Comment (pncifra) [inline src/components/Profile.tsx]: new profile component

File: src/components/Scoreboard.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
```
Comment (pncifra) [inline src/components/Scoreboard.tsx]: new scoreboard component, similar to profile

File: App.tsx [hunk 3]
```diff
@@ -1,19 +1,37 @@
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
```
Comment (pncifra) [inline App.tsx]: updated main app to include navigation to the new components

**Comment by VeronikaHar** [comment]
Merge

**Comment by VeronikaHar** [comment]
Merge

**Pull request merged**